zeth-contracts/contracts/BaseMixer.sol

```diff
@@ -57,9 +56,15 @@ contract BaseMixer is MerkleTreeMiMC7, ERC223ReceivingContract {
     uint constant jsOut = 2; // Nb of commitments/ciphertexts
 
     // We have 2 field elements for each digest (nullifierS (jsIn), commitmentS (jsOut), h_iS (jsIn) and h_sig)
-    // The root, v_pub_in and v_pub_out are all represented by one field element, so we have 1 + 1 + 1 extra public values
+    // The root, v_pub_in and v_pub_out are all represented by one field element, so we have 1 extra public values
     uint constant nbInputs = 1 + 2 * (jsIn + jsOut) + 1 + 1 + 2 * (1 + jsIn);
 
+    // Constants regarding the hash digest length, the prime number used and its associated length in bits and the max values (v_in and v_out)
+    uint constant digest_length = 256;
+    uint r = 21888242871839275222246405745257275088548364400416034343698204186575808495617;
```

Comment:
`r` is not used. Only its bit length should be required to get the bit-length of scalar field elements and parse the public inputs received here.